cli/src/helpers/mpl-token-auth-rules.ts

```diff
@@ -44,37 +44,40 @@ export const validateOperation = async (
     connection: Connection,
     payer: Keypair,
     name: string,
+    mint: PublicKey,
     operation: string,
     payload: Payload,
 ) => {
 
-    let op_type: Operation = Operation.Transfer;
+    let op_type: string = "0";
     switch (operation) {
         case "Transfer":
-            op_type = Operation.Transfer;
+            op_type = "0";
             break;
         case "Delegate":
-            op_type = Operation.Delegate;
+            op_type = "1";
             break;
         case "SaleTransfer":
-            op_type = Operation.SaleTransfer;
+            op_type = "2";
             break;
         case "MigrateClass":
-            op_type = Operation.MigrateClass;
+            op_type = "3";
             break;
```

Comment:
I don't understand why the string is being turned into a string of a number.  Am I missing something basic after Operation was stringified?

Reply:
My bad, I thought for some reason this is how we're doing it but I'll switch.